mithril-aggregator/src/http_server/routes/certificate_routes.rs

```diff
@@ -60,6 +73,24 @@ mod handlers {
         }
     }
 
+    /// List all Certificates
+    pub async fn certificate_certificates(
+        certificate_store: Arc<CertificateStore>,
```

Comment:
This should use the open_message service to get a list of certificates.

Reply:
Done :heavy_check_mark: the route now uses the `CertifierService`